WordPress/src/main/res/layout/stats_block_text_item.xml

```diff
@@ -1,19 +1,39 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
     android:id="@+id/container"
     style="@style/StatsBlockLine"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:minHeight="@dimen/one_line_list_item_height"
-    android:paddingStart="@dimen/margin_extra_large"
-    android:paddingEnd="@dimen/margin_extra_large">
+    android:minHeight="@dimen/one_line_list_item_height">
 
     <com.google.android.material.textview.MaterialTextView
         android:id="@+id/text"
         style="@style/StatsBlockText"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:paddingStart="@dimen/margin_extra_large"
```

Comment:
No need to add paddingStart for TextView, because the parent `ConstraintLayout` has already padding start `16dp`.